packages/models/src/lib/plugins.ts

```diff
@@ -0,0 +1,116 @@
+import { z } from 'zod';
+
+// Define Zod schema for the PluginMetadata type
+const pluginMetadataSchema = z.object(
+  {
+    slug: z.string({
+      description: 'Unique ID (human-readable, URL-safe)',
+    }),
+    name: z.string({
+      description: 'Display name',
+    }),
+    type: z.enum(
+      [
+        'static-analysis',
+        'performance-measurements',
+        'test-coverage',
+        'dependency-audit',
+      ],
+      { description: 'Plugin categorization' },
+    ),
```

Comment:
I don't think we have a usage for this after all, so let's leave it out (YAGNI principle).

```suggestion
```

Reply:
love it!